hack/owner_reset.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+sudo chown -R $USER bin \
+    chown $USER .git/index
+    
```

Comment:
Where is this used?

Reply:
This script resolves permission issues which can be encountered when using podman after using with docker.